package.json

```diff
@@ -28,6 +28,7 @@
     "sass": "^1.32.2",
     "semver": "^7.3.4",
     "sudo-prompt": "^9.2.1",
+    "t": "^0.5.1",
```

Comment:
We don't seem to be using this in this patch?

Reply:
I don't know how that got in there, and it didn't survive the latest rebase against `main`. But if we want to pull in modules from `dashboard` without changing them, we'll need to implement it anyway.